### DISCUSSION TITLE
Does Prisma works good with ESM?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#4623] [Q&A] [answered]
Does Prisma works good with ESM?

**Comment by ryands17** [comment, marked as answer]
Hey @cuBeUrg 👋 
By ESM, do you mean ESModules, or is it something else?

**Reply by ghost** [reply]
Hi! I mean ESModules.

**Reply by nikolasburk** [reply]
Can you maybe elaborate a bit more on your question and what exactly you're trying to achieve! In general, Prisma does work very well with ESModules though if that's just your main general question 🙂

**Reply by ghost** [reply]
Some of the ORM out there, don’t provide all the imports for ESM.

**Reply by nikolasburk** [reply]
Ah I see, do you have any specific concerns about features that you'd like to use? 🙂 Generally Prisma is implemented with the latest Node.js and TypeScript technologies, so I don't think there should be any issues. Nonetheless, if you have any specific questions feel free to let me know, happy to dig into this a bit further with you if needed! 🙌

**Reply by ghost** [reply]
Thanks!
